fs_storage/models/fs_storage.py

```diff
@@ -130,6 +130,16 @@ def __init__(self, env, ids=(), prefetch_ids=()):
         help="Relative path to the directory to store the file"
     )
 
+    store_connection = fields.Boolean(
+        default=True,
+        help="If marked, the connection to the filesystem is stored in memory.",
+    )
+    # https://filesystem-spec.readthedocs.io/en/latest/api.html#fsspec.spec.AbstractFileSystem.clear_instance_cache  # noqa: B950
+    invalidate_cache_on_error = fields.Boolean(
+        help="If marked, the filesystem cache will be invalidated when there is an "
+        "error in the connection. Invalidating the cache will force a reconnection.",
```

Comment:
@JordiMForgeFlow  Should this be set to True by default to ensure a safe use of fspecs?

Reply:
@lmignon I agree, not even sure if there will be cases where this is not wanted, but we can keep it as a setting.